paper_sections/02-gams.Rmd

```diff
@@ -1,33 +1,39 @@
 $$
+where $\mathbb{E}(y)$ is the expected value of the response $y$ (with an appropriate distribution and link function $g$), $f_j$ is a smooth function of the covariate $x_j$, $\beta_0$ is an intercept term and $g^{-1}$ is the inverse link function. Here there are $J$ smooths and each is a function of only one covariate, though it is possible to construct smooths of multiple variables.
 
-where $y$ is the response (with an appropriate distribution and link function $g$), $f_j$ is a smooth function of the covariate $x_j$, $\beta_0$ is an intercept term and $g^{-1}$ is the inverse link function. Here there are $J$ smooths and each is a function of only one covariate, though it is possible to construct smooths of multiple variables.
-
-Each smooth $f_j$s is represented by a sum of simpler *basis functions* ($b_k$) multiplied by corresponding coefficients ($\beta_k$), which need to be estimated to be estimated:
-
+Each smooth $f_j$ is represented by a sum of simpler *basis functions* ($b_k$) multiplied by corresponding coefficients ($\beta_k$), which need to be estimated:
 $$
-f_j(x_j) = \sum_{k=1}^K \beta_k b_k(x_j),
+f_j(x_j) = \sum_{k=1}^K \beta_k b_k(x_j)\,,
 $$
+The size $K$ of each smooth will determine the complexity of the resulting term (referred to as "basis size", "basis complexity" or "basis richness"). Though it seems like the basis can be overly complex ("how big should I make $K$?") and lead to overfitting, we need not worry about this as we use a penalty to ensure that each function's complexity is appropriate; hence the basis only need be *large enough* and we let the penalty deal with excess wigglyness.
 
-The size of $K$ of each smooth will determine the flexibility of the resulting term (referred to as "basis size", "basis complexity" or "basis richness"). Though it seems like the basis can be overly complex ("how big should I make $K$?") and lead to overfitting, we need not worry about this as we use a penalty to ensure that the functions complexity is appropriate; hence the basis only need to be "large enough" and we let the penalty deal with excess wigglyness.
+The penalty for a term is usually based on a derivative of the smooth, as the derivative is used to measure the wigglyness of the function and hence its flexibility. Model fit is measured by its likelihood ($\mathcal{L}(y|\theta)$, the likelihood of observing the data, $y$, given the parameters $\theta$). We trade-off the fit of the model against the wigglyness penalty to obtain a model that fits the data well but does not overfit. To control this trade-off we estimate a *smoothness parameter* for each smooth $\lambda_j$. This trade-off results in the penalised likelihood $\mathcal{L}_p$ of the estimated model
+$$
+\mathcal{L}_p = \underbrace{\mathcal{L}(y | \theta)}_{\text{model fit}} + \lambda \underbrace{\beta^{\mathsf{T}}\mathbf{S}\beta}_{\makebox[0pt]{\text{{\scriptsize wiggliness}}}}.
```

Comment:
either need to specify that this is a model with a single smooth with a single smoopar or write this more generally -- I'd side with the former?

Reply:
I'd be happy to stick with the former and I'm sure our readers will too. Given this, perhaps the previous comment doesn't need addressing and we can state this is for a single smooth with a single smoothness parameter and not refer to lambda_j before but just use lambda.

Reply:
Yeah, this makes things much easier (sorry going through this in order, which I'm realising was a silly idea...)